static/app/components/tours/components.tsx

```diff
@@ -307,7 +317,7 @@ export function TourGuide({
                 animated
                 arrowProps={{...arrowProps, background: 'lightModeBlack'}}
               >
-                <TourBody id={id}>
+                <TourBody ref={setScrollElement}>
```

Comment:
what i've seen people doing these days is just using the ref callback to trigger scroll instead of state + useEffect.
```tsx
   // somewhere outside component
   function scrollToElement(el: HTMLDivElement | null) {
	   if (el) {
	     el.scrollIntoView?.({block: 'center', behavior: 'smooth'});
	   }
   }
                <TourBody ref={scrollToElement}>
```

Reply:
Wow that's super clever, I like it 👍